server/broadcast.go

```diff
@@ -376,10 +377,12 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string,
 	}
 	res, err := SubmitSegment(sess, seg, nonce)
 	if err != nil || res == nil {
+		glog.Error("unable to submit segment err=", err)
 		cxn.sessManager.removeSession(sess)
 		if res == nil && err == nil {
-			return nil, err
+			err = errors.New("empty response")
```

Comment:
Ah so we used to return `errors.New("empty response")` here and I think it was accidentally changed in a previous PR - 👍 to switch back to returning `errors.New("empty response")` here.

Reply:
yeah exactly. Could have done it in a seperate PR but I felt like it was somewhat related and the diff is rather small.